target/linux/ath79/image/generic.mk

```diff
@@ -664,6 +664,20 @@ define Device/rosinson_wr818
 endef
 TARGET_DEVICES += rosinson_wr818
 
+define Device/trendnet_tew-823dru
+  ATH_SOC := qca9558
+  DEVICE_TITLE := Trendnet TEW-823DRU
```

Comment:
Please update to use DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT as introduced in:
https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5